### DISCUSSION TITLE
Rotated lat/lon grid using python embedding

### DISCUSSION THREAD

**Discussion by CPKalb** [dtcenter/METplus#1549] [File I/O] [answered]
Hi:

Is it possible to read in a rotated lat/lon grid into MET using python embedding?  If it is possible, is there an example of what grid variables are needed?

Thanks,
Tina

**Comment by JohnHalleyGotway** [comment, marked as answer]
@CPKalb yes, it **should** be supported. I see rotated lat/lon grids listed in the description of grid specification strings in [Appendix B ](https://met.readthedocs.io/en/latest/Users_Guide/appendixB.html#grid-specification-strings)of the MET user's guide. And [Appendix F](https://met.readthedocs.io/en/latest/Users_Guide/appendixF.html#python-embedding-for-2d-data) lists out the elements which define rotated lat/lon grids for python embedding:

- Rotated LatLon grid dictionary entries:
  - type (“Rotated LatLon”)
  - name (string)
  - rot_lat_ll, rot_lon_ll (double)
  - delta_rot_lat, delta_rot_lon (double)
  - Nlat, Nlon (int)
  - true_lat_south_pole, true_lon_south_pole (double)
  - aux_rotation (double)

While [unit_regrid.xml](https://github.com/dtcenter/MET/blob/main_v10.1/test/xml/unit_regrid.xml) does contain a "rotlatlon" grid specification string example, I do not see any examples of using it with python embedding.

Please give it a try, based on the documentation, and let us know if it doesn't work as expected.
